stempeg/read.py

```diff
@@ -385,3 +397,46 @@ def channels(self, idx):
     def __repr__(self):
         """Print stream information"""
         return pprint.pformat(self.audio_streams)
+
+
+def _read_mp4box_stem_titles(filename):
+    """Reads a mp4 stem titles file using MP4Box
+    Mainly taken from https://github.com/axeldelafosse/stemgen/blob/master/ni-stem/_internal.py
+    """
+    if not mp4box_exists():
+        warnings.warn(
```

Comment:
I like that the use of mp4box is optional but maybe it's better to not warn per se

Reply:
That makes sense, I will remove the warnings.

Reply:
Made the changes.